charts/aws-ebs-csi-driver/templates/volumesnapshotclass.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if or .Values.sidecars.snapshotter.forceEnable (.Capabilities.APIVersions.Has "snapshot.storage.k8s.io/v1beta1") (.Capabilities.APIVersions.Has "snapshot.storage.k8s.io/v1") }}
+{{- range .Values.volumeSnapshotClasses }}
+---
+kind: VolumeSnapshotClass
+apiVersion: snapshot.storage.k8s.io/v1
+metadata:
+  name: {{ .name }}
+  {{- with .annotations }}
+  annotations: {{- . | toYaml | trim | nindent 4 }}
+  {{- end }}
+driver: ebs.csi.aws.com
+deletionPolicy: {{ .deletionPolicy }}
+{{- end }}
+{{- end }}
```

Comment:
Minor nit, but could we make sure this file has a newline at the end?

Reply:
The review UI won't let you see this but if you click over to the "files changed" tab you should see the issue.